boilerplate/templates/styles/code-snippet.scss

```diff
@@ -0,0 +1,80 @@
+/*
+Monokai style
```

Comment:
I think a better place for this would be `/frontend/scss/`.

Reply:
It would indeed be a better place, but I forgot that the boilerplate generator uses it own build. So, please ignore and leave things as they are :-).

Reply:
So should I revert the changes completely or just leave them in the state I submitted yesterday? :-)